metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -1086,9 +1093,12 @@ def get_upstream_columns_of_fields_in_datasource(
                     and upstream_table_id
                     and upstream_table_id in table_id_to_urn.keys()
                 ):
+                    parent_dataset_urn = table_id_to_urn[upstream_table_id]
+                    if self.is_snowflake_urn(parent_dataset_urn):
+                        name = name.lower()
```

Comment:
fix for tableau native CLL with snowflake upstream

Reply:
could you add a comment explaining that the snowflake source lowercases column names, so this behavior is to match that?

Reply:
done.